homeassistant/components/accuweather/strings.json

```diff
@@ -24,95 +24,38 @@
   },
   "entity": {
     "sensor": {
-      "pressure_tendency": {
-        "state": {
-          "steady": "Steady",
-          "rising": "Rising",
-          "falling": "Falling"
-        }
-      },
-      "air_quality": {
-        "state": {
-          "good": "Good",
-          "hazardous": "Hazardous",
-          "high": "High",
-          "low": "Low",
-          "moderate": "Moderate",
-          "unhealthy": "Unhealthy"
-        }
-      },
       "grass_pollen": {
         "state_attributes": {
           "level": {
-            "name": "Level",
```

Comment:
Interesting. I don’t know if this an improvement ? For the user in the UI nothing changes, but things like blueprints will no longer work, as the state is no longer a standard value.

Reply:
To be honest, I'm not entirely happy with this change myself. Several users asked to add language support, so I decided to open PR and wait for feedback. The main advantage of this change is that the state of the `condition` sensors will be in selected language. We are not able to translate these states because they are random sentences. The disadvantage of this change is that it breaks the `air quality` sensor states and `level` attributes. Unfortunately, the API does not return values in the selected language and in English in one payload, and we can't use more requests because we don't have enough of them available.
What do you think? Is it worth working on this further, or do we reject such a change?

Reply:
I am leaning to rejecting this. We’re trying to standardize things in core, and allow localization to be done in the frontend.

Reply:
I have to agree with Paulus on this one. 

The state itself is technical implantation detail, the provided translations for those states are how they are represented.

This PR is diverting from that, making the result unexpected/inconsistent. For example, if an user would change their language, their existing automations relying on this will break.

../Frenck